client/platform/desktop/frontend/components/ImportDialog.vue

```diff
@@ -37,10 +46,15 @@ export default defineComponent({
 
     const sortedFpsOptions = computed(() => {
       const filteredOptions = FPSOptions
-        .filter((v) => v <= Math.round(argCopy.value.jsonMeta.originalFps));
-      filteredOptions.push(argCopy.value.jsonMeta.originalFps);
-      filteredOptions.sort((a, b) => a - b);
-      return uniq(filteredOptions);
+        .filter((v) => v.value <= Math.round(argCopy.value.jsonMeta.originalFps));
+      const videoFrameRateIndex = filteredOptions.findIndex((item) => item.value === -1);
+      if (videoFrameRateIndex !== -1) {
+        filteredOptions[videoFrameRateIndex] = {
+          text: `Video Frame Rate (${argCopy.value.jsonMeta.originalFps})`,
+          value: argCopy.value.jsonMeta.originalFps,
+        };
+      }
+      return filteredOptions;
```

Comment:
Could you put the sort back in?  `60` appears before `59.97` in the list if 59.97 is the original.

Reply:
could this be better fixed by eliminating the `Math.round` from the annotation filter.  You shouldn't be able to choose 60 if your video frame rate is 59.97?

Reply:
Yes, I think that's right.